packages/@webex/plugin-cc/src/services/core/WebSocket/WebSocketManager.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable */
```

Comment:
For some reason, it was showing an error on the console.error hence, disabled it.
But overall, this class is a port of the `aqm-notifs.ts` from agentx. It performs the same functions, although it has been refactored slighly based on the cc-sdk use cases.
Overall it performs the same as well, and all the logic including worker thread, keep alive and the various web socket handling is exactly the same as in agent desktop.